src/test/java/org/folio/innreach/fixture/MappingFixture.java

```diff
@@ -54,6 +56,20 @@
     libraryAndLocationRandom = new EasyRandom(params);
   }
 
+  static {
+    EasyRandomParameters params = new EasyRandomParameters()
+      .randomize(named("patronType"), new IntegerRangeRandomizer(0, 256))
```

Comment:
shouldn't it be 255 instead to match DB constraint?

Reply:
No, since in the IntegerRangeRandomizer constructor max value is exclusive, meaning that the code above will create an object that will produce values in a range of [0, 255].